SourceX/controls/devices/kbcontroller.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+// Keyboard keys acting like gamepad buttons
+#ifndef HAS_KBCTRL
+#define HAS_KBCTRL 0
+#endif
+
+#if defined(RETROFW)
+#define HAS_KBCTRL 1

```

Comment:
Shoudn't this go in the CMakeList.txt file?

Reply:
It's up to you: I can move to CMakeLists.txt if you prefer
Alternatively, we can even maybe specify this from the build script for RetroFW by passing arguments to cmake

Reply:
Now defined via the build script (by passing arguments to cmake)!